package.json

```diff
@@ -91,28 +85,20 @@
     "@vitest/coverage-v8": "^3.1.2",
     "@vitest/ui": "^3.1.2",
     "babel-plugin-transform-imports": "git+https://git@github.com/fabricjs/babel-plugin-transform-imports.git",
-    "busboy": "^1.6.0",
```

Comment:
many deps now removed
fabric is now really a more lightweight project in terms of maintenance